src/components/ClickableStyle/ClickableStyle.tsx

```diff
@@ -0,0 +1,134 @@
+import clsx from 'clsx';
+import React, { MouseEventHandler, ReactNode } from 'react';
+import styles from './ClickableStyle.module.css';
+import Icon from '../Icon';
+
+export type ClickableStyleProps<IComponent extends React.ElementType> = {
+  /**
+   * Visually hidden clickable text (but text is still accessible to assistive technology).
+   * This overrides `children` for screen readers
+   */
+  'aria-label'?: string;
+  /**
+   * CSS class names that can be appended to the component
+   */
+  className?: string;
+  /**
+   * Disables the field and prevents editing the contents
+   */
+  disabled?: boolean;
+  /**
+   * ClickableStyle reference
+   */
+  forwardRef?: any;
```

Comment:
why do we need the `forwardRef` prop? wouldn't people always pass `ref`

Reply:
This was copied over from `Button`. I don't think we need it; removing.